microsoft-edge/webview2/how-to/Print.md

```diff
@@ -0,0 +1,755 @@
+
+            return S_OK;
+          })
+          .Get()));
+  return true;
+}
+```
+
+**APIs:**
+
+* [ICoreWebView2_16::Print method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#print)
+* [ICoreWebView2Environment6::CreatePrintSettings method](/microsoft-edge/webview2/reference/win32/icorewebview2environment6#createprintsettings)
+* [ICoreWebView2PrintSettings interace](microsoft-edge/webview2/reference/win32/icorewebview2printsettings)
+* [ICoreWebView2PrintSettings2 interace](microsoft-edge/webview2/reference/win32/icorewebview2printsettings2)
+* [COREWEBVIEW2_PRINT_STATUS enum](/microsoft-edge/webview2/reference/win32/icorewebview2_16#corewebview2_print_status)
+
+---
+
+<!-- end of tab-set -->
+
+
+<!-- ====================================================================== -->
+## The PrintToPdf method to customize printing to a PDF file
+
+Silently prints the current top-level document in the WebView2 control to a PDF file. To completely control how printing is performed, you can print to a PDF and then build your own code to print the PDF.
+
+This API consists of an asynchronous `PrintToPdf` method and a `PrintSettings` object. The `PrintToPdf` method accepts a path that the PDF file will be saved to.
+
+<!-- ------------------------------ -->
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2.PrintToPdfAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printtopdfasync#microsoft-web-webview2-core-corewebview2-printtopdfasync)
+* [CoreWebView2Environment.CreatePrintSettings Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createprintsettings)
+* [CoreWebView2PrintSettings Class](/dotnet/api/microsoft.web.webview2.core.corewebview2printsettings)
+
+<!-- ------------------------------ -->
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2.PrintToPdfAsync Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#PrintToPdfAsync)
+* [CoreWebView2Environment.CreatePrintSettings Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createprintsettings)
+* [CoreWebView2PrintSettings Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2printsettings)
+
+<!-- ------------------------------ -->
+
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2_7::PrintToPdf method](/microsoft-edge/webview2/reference/win32/icorewebview2_7#printtopdf)
+* [ICoreWebView2Environment6::CreatePrintSettings method](/microsoft-edge/webview2/reference/win32/icorewebview2environment6#createprintsettings)
+* [ICoreWebView2PrintSettings interface](microsoft-edge/webview2/reference/win32/icorewebview2printsettings)
+
+---
+
+<!-- end of tab-set -->
+
+#### Example: The PrintToPdf method to customize printing to a PDF file
+
+This example prints the current webpage to a PDF file, using the default path and settings.
+
+<!-- ------------------------------ -->
+
+##### [.NET/C#](#tab/dotnetcsharp)
```

Comment:
maybe this is a good line and an old line is bad.

doc build error:
Line 519, Column 1: [Warning: invalid-tab-group - See documentation] Tab group with different tab id set.